apps/docs/src/common/pages/get-started.tsx

```diff
@@ -1,10 +1,40 @@
 import { FC, Fragment, createElement as h } from 'react';
 import { PageProps } from '@not-govuk/app-composer';
+import { A } from '@not-govuk/components';
 
-import Markdown from '../../../../../docs/get-started.md';
 
-const Page: FC<PageProps> = props => (
-  <Markdown />
+const Page: FC<PageProps> = ({ location }) => (
+  <div className="govuk-grid-row">
+    <div className="govuk-grid-column-one-third">
+      <aside>
+      <h2>Get started</h2>
+        <ul className="plain">
+          <li><A href="/get-started/prototype">Prototyping for designers</A></li>
+          <li><A href="/get-started/project">Starting a project for developers</A></li>
```

Comment:
Maybe we should keep it simple here and explain more in the text on the right. We could follow the GDS pattern here. i.e.
- prototyping
- production code